.gitmodules

```diff
@@ -81,3 +81,6 @@
 [submodule "vendor/tree-sitter-elm"]
 	path = vendor/tree-sitter-elm
 	url = https://github.com/Razzeee/tree-sitter-elm.git
+[submodule "vendor/tree-sitter-clojure"]
+	path = vendor/tree-sitter-clojure
+	url = https://github.com/sogaiu/tree-sitter-clojure
```

Comment:
Looks like `.git` is missing at the end in comparison with lines above.

Reply:
oh no.  I will correct this now

Reply:
wait, I thought these lines were added directly from using the git submodule command.  I didn't hand-enter these line.

Reply:
looks like all the other entries contain the .git type, so I updated the .gitmodules file.  I probably did something wrong.